addon/components/cfb-form-editor/question-list/item.hbs

```diff
@@ -68,17 +68,26 @@
     {{/if}}
 
     <span class="uk-position-relative uk-width-auto">
-      <UkBadge
-        class="cfb-form-editor__question-list__item__type"
-        @label={{t (concat "caluma.form-builder.question.types." @question.__typename)}}
-      />
-      {{#if this.required}}
-        <span
-          title={{t "caluma.form-builder.question.isRequired"}}
-          class="uk-position-top-right cfb-form-editor__question-list__item__required"
-        >
-        </span>
-      {{/if}}
+      {{#let 
+        (if this.required (if (eq this.required true) "required" "conditional") "not-required")
+        (if this.hidden (if (eq this.hidden true) "hidden" "conditional") "not-hidden")
+        as |requiredType hiddenType|
+      }}
+        <UkBadge
+          uk-tooltip="title: {{if this.hidden (t (concat "caluma.form-builder.question-list.hidden." hiddenType))}}; pos: left"
+          class="{{if this.hidden (concat "cfb-form-editor__question-list__item__hidden")}} cfb-form-editor__question-list__item__type"
+          @label={{t (concat "caluma.form-builder.question.types." @question.__typename)}}
+          title={{if this.hidden (t (concat "caluma.form-builder.question-list.hidden." hiddenType))}}
+        />
+        {{#if this.required}}
+          <span
+            uk-tooltip="title: {{t (concat "caluma.form-builder.question-list.required." requiredType)}}; pos: top-left"
+            title={{t (concat "caluma.form-builder.question-list.required." requiredType)}}
+            class="uk-position-top-right cfb-form-editor__question-list__item__required-{{requiredType}}"
```

Comment:
```suggestion
            class="uk-position-top-right cfb-form-editor__question-list__item__required cfb-form-editor__question-list__item__required--{{requiredType}}"
```

`ember-caluma` uses [BEM](http://getbem.com) as CSS convention. Feel free to ask me about it if you want to know more - it's hard to put it into a comment :wink: